sdk/ml/azure-ai-ml/azure/ai/ml/entities/_monitoring/signals.py

```diff
@@ -129,16 +217,45 @@ def __init__(
         )
         self.type = MonitorSignalType.PREDICTION_DRIFT
 
+    def _to_rest_object(self) -> RestPredictionDriftMonitoringSignal:
+        return RestPredictionDriftMonitoringSignal(
+            # hack this to a random value since the service ignores it and it's mislabled as a required property
```

Comment:
Is this going to be fixed soon on the service side, like in a new API version?

Reply:
yeah, the MoMo service folks only realized this issue after the ARM contract close. I think it's been fixed in the next API version